keyboards/xd75/keymaps/hvp/keymap.c

```diff
@@ -0,0 +1,119 @@
+/* Copyright 2017 Lukmanul Hakim
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "hvp.c"
+
+// Layer shorthand
+#define _QW 0
+#define _L1 1
+#define _L2 2
+#define _L3 3
+
+#define LT3_ESC LT(3, KC_ESC)
+#define LT4_TAB LT(4, KC_TAB)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* SPLIT QWERTY
+ * .--------------------------------------------------------------------------------------------------------------------------------------.
+ * | ESC    | 1      | 2      | 3      | 4      | 5      | NUM/   | NUM*   | NUM-   | 6      | 7      | 8      | 9      | 0      | `      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------|
+ * | TAB    | Q      | W      | E      | R      | T      | NUM7   | NUM8   | NUM9   | Y      | U      | I      | O      | P      | BSPACE |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+ * | LCTRL  | A      | S      | D      | F      | G      | NUM4   | NUM5   | NUM6   | H      | J      | K      | L      | ;      | ENTER  |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+ * | LSHIFT | Z      | X      | C      | V      | B      | NUM1   | NUM2   | NUM3   | N      | M      | <      | >      | /      | RSHIFT |
+ * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+ * | LALT   | RAISE  | RAISE  | LGUI   | SPACE  | SPACE  | NUM+   | NUM0   | NUM.   | SPACE  | SPACE  | RGUI   | LOWER  | LOWER  | RALT   |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+	
+ [_QW] = { /* SPLIT QWERTY */
```

Comment:
Please use the `LAYOUT_ortho_5x15` macro here.

Reply:
Please ignore, still work in progress

Reply:
I moved it to a different branch.